axon-server/installation/docker-k8s/axon-server-se.md

```diff
@@ -0,0 +1,278 @@
+    "default"
+  ],
+  "httpPort": 8024,
+  "grpcPort": 8124,
+  "internalHostName": null,
+  "grpcInternalPort": 0,
+  "name": "{axon-server-name}",
+  "hostName": "{axon-server-hostname}"
+}
+```
+
+The application is installed in the root with a minimal properties file which is depicted below. The “/data” and “/eventdata” directories are created as volumes, and their data will be accessible on the local filesystem somewhere in Docker’s temporary storage tree.axoniq.axonserver.event.storage=/eventdata
+
+```text
+axoniq.axonserver.snapshot.storage=/eventdata
+axoniq.axonserver.controldb-path=/data
+axoniq.axonserver.pid-file-location=/data
+logging.file=/data/axonserver.log
+logging.file.max-history=10
+logging.file.max-size=10MB
+```
+
+### Customization
+
+The directory locations for the volumes can be specified as per your requirements. The image also has a third directory "/config" which is not marked as a volume. This gives you the capability to have an "axonserver.properties" file which can be placed in this location to override the above mentioned settings as well as add new properties similar to a local install.
+
+Assuming that you have a directory "_axonserverse**"**_ which will be the designated location for your volumes and configuration information.
+
+We will first create the sub-directories for the volumes/configuration. We will also add a couple of custom properties \(name/hostname\) to the axonserver.properties file which will be placed in the config sub-directory. As stated above, you can add additional properties to control the configuration.
+
+```text
+$ mkdir -p axonserverse/data axonserverse/events axonserverse/config
+$ (
+> echo axoniq.axonserver.name=axonserver
+> echo axoniq.axonserver.hostname=localhost
+> ) > axonserverse/config/axonserver.proprties
+```
+
+To start the container with the customizations done above, the following command can be executed
+
+```text
+docker run -d --rm --name axonserver -p 8024:8024 -p 8124:8124 -v `pwd`/axonserverse/data:/data -v `pwd`/axonserverse/events:/eventdata -v `pwd`/axonserverse/config:/config axoniq/axonserver
+```
+
+Now if you query the API \(utilizing the “curl” command depicted above\) it will show that it is running with name “axonserver” and hostname “localhost”. Also the data directory will contain the ControlDB file, PID file, and a copy of the log output. The “events” directory will have the event and snapshot data.
+
+This completes a basic setup of the Axon Server SE Docker image with implementation of customizations.
+
+## Docker Compose
+
+Running Axon Server SE in docker-compose helps address more complex requirements around distributed scenarios. The following file will help start Axon Server SE with “./data”, “./events”, and “./config” mounted as volumes and the config directory is actually Read-Only.
+
+This again assumes that you have a directory "_axonserverse**"**_ which will be the designated location for your volumes and configuration information.
```

Comment:
The same "What is the purpose of the double-star notation?" from @bert-laverman would apply here.

Reply:
Italics